report_qweb_decimal_place/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,27 @@
+This module intends to provide the base function for currencies to adjust the number of decimal places
+for the unit price in QWeb reports.
+Installing this module alone does not affect the presentation of existing QWeb reports.
+Individual adjustments need to be done in separate modules in a manner similar to the following:
+
+.. code-block:: xml
+
+    <template
+        id="report_saleorder_document_inherit"
+        inherit_id="sale.report_saleorder_document"
+    >
+        <xpath expr="//td[@name='td_priceunit']/span" position="replace">
+                <t t-set="currency" t-value="doc.currency_id" />
+                <t t-set="price_unit" t-value="line.price_unit" />
+                <t t-call="report_qweb_decimal_place.price_unit_value_format" />
+        </xpath>
+    </template>
+
+Background:
+-----------
```

Comment:
```suggestion
~~~~~~~~~~~

```
Pay attention to the blank line as well.

ref. https://github.com/OCA/maintainer-tools/pull/455#issuecomment-636305924